pkg/platform/api/mono/mono.go

```diff
@@ -34,7 +37,32 @@ func Init(serviceURL *url.URL, auth *runtime.ClientAuthInfoWriter) *mono_client.
 	if auth != nil {
 		transportRuntime.DefaultAuthentication = *auth
 	}
-	return mono_client.New(transportRuntime, strfmt.Default)
+	client := mono_client.New(transportRuntime, strfmt.Default)
+
+	// For the Oauth client, prefer use of IPv4. This is needed particularly for device
+	// authorization, which compares the IP address of the State Tool request and the IP address of
+	// the web client request. If there's a mismatch, authorization fails. When this happens, it's
+	// often because the State Tool connects to the Platform via IPv6, but the browser does via IPv4
+	// (browsers apparently prefer IPv4 for now).
+
+	httpTransport, ok := http.DefaultTransport.(*http.Transport)
+	if !ok {
+		return client // for some reason the assertion can fail...
```

Comment:
I think this is a good example of a rare case in which we want a panic. This shouldn't ever happen, and if we put a panic here we're sure to know that it is happening long before we put out a release with this issue.

Reply:
It does panic during some unit tests in the project package. I don't understand why, as this is part of Go's stdlib. I can try to spend more time looking into it, but I'm worried about rabbit holes.

Reply:
Here's one of the panics: https://github.com/ActiveState/cli/runs/5414128742?check_suite_focus=true#step:9:79

Reply:
Oh... it's in a mock test. I think I found something to delete... Okay, we can panic then.